.circleci/config.yml

```diff
@@ -2,6 +2,7 @@ version: 2.1
 
 orbs:
   slack: circleci/slack@volatile
+  circle-compare-url: iynere/compare-url@volatile
```

Comment:
We should probably avoid volatile for this.

Reply:
@ndintenfass good point